src/main/kotlin/no/nav/familie/ks/sak/api/dto/ForhåndsvisTilbakekrevingsvarselbrevDto.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.familie.ks.sak.api.dto
+
+data class ForhåndsvisTilbakekrevingsvarselbrevDto(val fritekst: String)
```

Comment:
```suggestion
data class ForhåndsvisTilbakekrevingsVarselbrevDto(val fritekst: String)
```
Nitpick, Camelcasing på varselbrev

Reply:
endret til ForhåndsvisTilbakekrevingVarselbrevDto